commands/info.js

```diff
@@ -1,14 +1,14 @@
 'use strict'
 
-let co = require('co')
-let cli = require('heroku-cli-util')
-let _ = require('lodash')
-let url = require('url')
-let host = require('../lib/host')
+const co = require('co')
+const cli = require('heroku-cli-util')
 
 function databaseNameFromUrl (uri, config) {
+  const invert = require('lodash.invert')
```

Comment:
Out of curiosity, is this some require magic, or does lodash actually ship micro-modules so you can avoid requiring the whole thing?